AppInspector.RulesEngine/OatExtensions/OatRegexWithIndexOperation.cs

```diff
@@ -83,7 +83,7 @@ private static IEnumerable<Violation> RegexWithIndexValidationDelegate(CST.OAT.R
         if (state1 is TextContainer tc && clause is OatRegexWithIndexClause src &&
             clause.Data is List<string> RegexList && RegexList.Count > 0)
         {
-            RegexOptions regexOpts = Common.Utils.RegexModifierToRegexOptions(src.Arguments);
+            RegexOptions regexOpts = Common.Utils.RegexModifierToRegexOptions(src.Arguments, enableNonbacktrackingRegex: src.Arguments.Contains("nb"));
```

Comment:
I think this is not necessary, since `nb` is added to the arguments when the pattern is added to the abstract rule set - as long as the `nb` and `b` cases are added to `RegexModifierToRegexOptions`